src/client.js

```diff
@@ -1436,15 +1436,16 @@ MatrixClient.prototype.exportRoomKeys = function() {
  * Import a list of room keys previously exported by exportRoomKeys
  *
  * @param {Object[]} keys a list of session export objects
+ * @param {Object} opts
```

Comment:
We've been using this style: https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/base-apis.js#L51 to document parameters passed in objects

Reply:
done